### PR TITLE
Add default icon to libnotify notifications

### PR DESCRIPTION
**Pull request by NeiRo21**

### CODE REVIEW

**Comment by maximbaz** [comment]
Thanks for the PR! It is kinda intentional that there's no built-in icon, you can drop your favorite icon into any standard icon location, name it `yubikey-touch-detector.png` and it will be picked up. I am not sold yet that we need to have something more than this, and especially to replace custom icons with a single hardcoded one.

**Comment by NeiRo21** [comment]
Makes sense. I think the only problem with this approach is that it's
unlikely to work out of the box, so it looks like libnotify integration is
broken. Besides there seems to be no mention of the setup required in the
readme.
What's your opinion on making notification icon configurable with the
Yubico logo as a fallback?

**Comment by maximbaz** [comment]
What if instead of specifying a path to the icon in XDG data, we keep `yubikey-touch-detector` as it is today (so no change in Go code), and simply install yubico icon into `/usr/share/icons/hicolor/128x128/apps/yubikey-touch-detector.png`? Then it would be automatically picked up, unless people put their override in `~/.local/share/icons/`.

File: service.conf.example [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# enable debug logging
+YUBIKEY_TOUCH_DETECTOR_VERBOSE=false
+
+# show desktop notifications using libnotify
+YUBIKEY_TOUCH_DETECTOR_LIBNOTIFY=true
+
+# print notifications to stdout
+YUBIKEY_TOUCH_DETECTOR_STDOUT=false
```
Comment (maximbaz) [inline service.conf.example:8]: this should be `true` to match the comment :wink:

File: service.conf.example [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+# disable Un*x socket notifier
+YUBIKEY_TOUCH_DETECTOR_NOSOCKET=false
```
Comment (maximbaz) [inline service.conf.example:11]: If you feel like it, could I ask you also to update https://github.com/maximbaz/yubikey-touch-detector#configuring-the-app with the missing config values? :pray:

**Comment by maximbaz** [comment]
Amazing, thanks so much! Looks good to me, should I merge, or is there anything else you want to do?

**Comment by AndrewKvalheim** [comment]
@NeiRo21 Did you see the existing discussion in #25? It links to an official touch icon that is probably a better fit than the Yubico Inc. logo.

> screenshot

**Pull request merged**